.env

```diff
@@ -0,0 +1,3 @@
+CLOUD_NAME = db3kljtw9
```

Comment:
This file should be added to .gitignore. Staging it breaks the purpose of using dotenv.

Reply:
I've already included it in the .gitignore file but I don't know why it's still visible

Reply:
https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore